hhsuite/hhsearch.xml

```diff
@@ -1,43 +1,123 @@
-<tool id="hhsearch" name="HHsearch" version="0.1.0" python_template_version="3.5">
+<tool id="hhsearch" name="HHsearch" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.01">
     <description>detecting remote homologues of proteins</description>
+    <macros>
+        <token name="@TOOL_VERSION@">3.2.0</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+        <xml name="ffindex_single_inputs">
+            <param name="ffdata" type="data" format="ffdata" label="PDB Database" help="Database Data file." />
```

Comment:
The label here should probably "HHM Database" and "HHM Database Index". Everything else looks great tho, thank you so much.